### PR TITLE
chore(readme): Add information to README

### PR DESCRIPTION
**Pull request by CyberFlameGO**
I've made a start on the readme. The README should be briefly informative in my opinion and we should make a contributing.md and security.md

### CODE REVIEW

**Comment by CyberFlameGO** [comment]
I forgot about this part lol

**Comment by CyberFlameGO** [comment]
Maybe merge this for now?

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 </h1>
 
 <h3 align="center">
-	Ender is a modular Discord bot that is extendable with plugins.
+	Ender is a highly configurable modular Discord bot that is extendable with plugins.
```
Comment (tjrgg) [inline README.md]: The bot isn't currently designed to be "highly configurable," just modular. I'd like to keep configuration rather simple, actually.

File: README.md [hunk 2]
```diff
@@ -63,9 +63,17 @@
 * GitHub: [@tjrgg](https://github.com/tjrgg)
 * Twitter: [@tjrgg](https://twitter.com/tjrgg)
 
+## Support
+Ender is not designed to run independently on your local machine and I will not provide support for "self-hosting". The project is designed to be exclusively for Ender, but open for developers and users to be able to contribute and open issues.
```
Comment (tjrgg) [inline README.md]: This shouldn't be written in the first person. Also, this disclaimer is already in the README under "License."

File: README.md [hunk 3]
```diff
@@ -63,9 +63,17 @@
...
+Support will be provided via GitHub issues if relevant to the publicly hosted bot, not self-hosting.
+
+## Contributions & Issues
```
Comment (tjrgg) [inline README.md]: I'm all for adding a contributing section, but not until there's something there.

File: README.md [hunk 4]
```diff
@@ -63,9 +63,17 @@
...
+
+<!-- TO-DO -->
 
 ## License
 
 Copyright © 2020 TJR Enterprise, LLC, Tyler Richards and contributors. All rights reserved.
 
 Use of the software and source code contained in this project is subject to the [Software License Agreement](LICENSE.md) (the "License") included with this project. The License is intended to allow you to access, download, modify and use the software and source code in this project for your personal benefit and to allow you to contribute to the project. The License is not intended to permit any other use of the software and source code in this project, including without limitation, commercial and proprietary uses.
+Also see [NOTICE.md](NOTICE.md).
```
Comment (tjrgg) [inline README.md]: I think this is redundant and unnecessary.

File: README.md (hunk 3 above)
Comment (CyberFlameGO) [reply README.md]: IK i was hoping one of yous would make a contributing.md

File: README.md (hunk 1 above)
Comment (CyberFlameGO) [reply README.md]: Oh right. Didn't know, I was just going off of the vibe in the Discord

File: README.md (hunk 3 above)
Comment (tjrgg) [reply README.md]: Yep, that's in the works. I'll be pushing it some time this week, probably, along with a Code of Conduct.

**Pull request closed without merging**